contracts/FanDonation.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: agpl-3.0
+
+pragma solidity ^0.6.0;
+
+import {IERC20} from "../contracts/interfaces/IERC20.sol";
+import {SafeERC20} from "../contracts/interfaces/SafeERC20.sol";
+
+import {ILendingPool} from "../contracts/interfaces/ILendingPool.sol";
+import {ILendingPoolAddressesProvider} from "../contracts/interfaces/ILendingPoolAddressesProvider.sol";
+
+contract FanDonation {
+    using SafeERC20 for IERC20;
+
+    // Set the address of the AAVE protocol addresses provider. Should not change once deployed.
+    // Currently set to the LendingPoolAddressesProvider for Kovan (https://docs.aave.com/developers/deployed-contracts)
+    ILendingPoolAddressesProvider public constant addressesProvider = ILendingPoolAddressesProvider(0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5);
+    ILendingPool lendingPool = ILendingPool(addressesProvider.getLendingPool());
```

Comment:
The linter will complain about not specifying the visibility for these state vars, maybe set them to private/internal